### ISSUE TITLE
Update procedure: breaking detection of ePC OS type?

### ISSUE THREAD

**Issue by KlausStrohhaecker** [nonlinear-labs-dev/C15#1974]
Issue detected:

- Check_preconditions() calls wait4playground() which seems to be intended to detect whether we have a NLL linux system on the ePC.
- This will only succeed when MS1.7 is already installed on the ePC.
- A MS1.5 ePC has no playground.service and thus this check will fail.
- Therefore, Check_preconditions() will try to reach the ePC assuming it is Windows system, which will fail also.

```
wait4playground() {
    for COUNTER in $(seq 1 $TIMEOUT); do
        echo "awaiting playground ... $COUNTER/$TIMEOUT"
        sleep 1
        executeAsRoot "systemctl status playground" && return 0
    done
    return 1
}

check_preconditions() {
    if ! wait4playground; then
        if [ -z "$EPC_IP" ]; then report "" "E81: Usage: $EPC_IP <IP-of-ePC> wrong ..." "Please retry update!" && return 1; fi
        if ! ping -c1 $EPC_IP 1>&2 > /dev/null; then  report "" "E82: Cannot ping ePC on $EPC_IP ..." "Please retry update!" && return 1; fi
        if executeOnWin "mountvol p: /s & p: & DIR P:\nonlinear"; then
            report "" "E84: Upgrade OS first!" "Please contact NLL!" && return 1
        else
            if mount.cifs //192.168.10.10/update /mnt/windows -o user=TEST,password=TEST \
                && ls -l /mnt/windows/ | grep Phase22Renderer.ens; then
                    report "" "E85: OS too old for update!" "Please contact NLL!" && return 1
            fi
        fi
        report "" "Something went wrong!" "Please retry update!" && return 1
    fi
    return 0
}
```

**Comment by KlausStrohhaecker** [comment]
Further, check_preconditions() tries to execute "mount.cifs" which is not installed on the BBB once it has been updated to MS1.5++.
check_preconditions() then breaks without producing a meaningful error message (which would be "cound not detect ePC OS" or something).

**Comment by KlausStrohhaecker** [comment]
fixed via #1958
